### PR TITLE
build: Release v37.0

### PR DESCRIPTION
**Pull request by likebreath**

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -310,6 +318,68 @@
...
+
+This release is a LTS release. Point releases for bug fixes will be made
+for the next 18 months; live migration and live upgrade will be
+supported between the point releases of the LTS.
+
+### Improved VFIO Device Passthrough with Multiple PCI Segments
+
+Now VFIO devices with 32-bit memory BARs can be attached to non-zero PCI
+segments on the guest, allowing users to assign such devices to
+appropriate NUMA nodes for better performance.
```
Comment (rbradford) [inline release-notes.md:337]: The use of segments also allows the addition of more than 31 devices.

**Review by rbradford — APPROVED** [review]

File: release-notes.md [hunk 2]
```diff
@@ -310,6 +318,68 @@
...
+
+This release is a LTS release. Point releases for bug fixes will be made
+for the next 18 months; live migration and live upgrade will be
+supported between the point releases of the LTS.
+
+### Multiple PCI segments Support for 32-bit VFIO devices
+
+Now VFIO devices with 32-bit memory BARs can be attached to non-zero PCI
+segments on the guest, allowing users to have more 32-bit devices and
+assign such devices to appropriate NUMA nodes for better performance.
```
Comment (likebreath) [inline release-notes.md:337]: @rbradford PTAL. When I started to rewrite this section and realize the discussion we had about the practical number of 32-bit devices we can use given the very limited 32-bit address space. (See https://github.com/cloud-hypervisor/cloud-hypervisor/pull/5953#issuecomment-1812916352).

For this reason I didn't mention anything specific about how many 32-bit devices we can use and keep it very general, e.g. `allowing users to have more 32-bit devices`. Please let me know if you have suggestions to improve it. Thanks.

Comment (rbradford) [reply release-notes.md:337]: Sounds great to me!

Comment (likebreath) [reply release-notes.md:337]: Great. Land this PR and cut the release now.

**Pull request merged**